app/src/models/item.ts

```diff
@@ -86,6 +87,11 @@ export class ItemModel {
         ? rawManifestMetadata["Division"].toString()
         : rawManifestMetadata["Library Locations"][0] || "";
 
+    this.permittedLocationText =
+      this.isRestricted && rawManifestMetadata["Permitted Locations"]
+        ? rawManifestMetadata["Permitted Locations"][0].toString()
```

Comment:
does the `toString` call need to be `?.toString()` in case the array is empty or null?

Reply:
I'll add